nevergrad/functions/fishing/test_core.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+from . import core
+
+
+def test_mixsimulator() -> None:
+    func = core.OptimizeFish()
+    x = np.random.rand(func.dimension)
+    value = func(x)  # should not touch boundaries, so value should be < np.inf
+    assert value < np.inf
```

Comment:
you should test against an actual value (possibly after seeding) to detect unintentional changes we could introduce

Reply:
Done, thx.